src/SearchIndex/index.ts

```diff
@@ -38,9 +38,10 @@ export class SearchIndex {
     this.convertedTaxonomy = {};
   }
 
-  async search(query: Query, searchProperty: string[] | null) {
+  async search(query: Query, searchProperty: string[] | null, pageNumber?: number) {
     const aggregationQuery = query.getAggregationQuery(searchProperty);
-    aggregationQuery.push({ $limit: 50 });
+    const RES_COUNT = 50;
+    const PAGINATED_RES_COUNT = 10;
     aggregationQuery.push({
```

Comment:
I think we decided on 250? Correct me if I'm wrong. And then maybe 50 a page... I believe.

Reply:
i will verify with design for results per page! `50 max limit` is whats currently existing (wont change), but a paginated requests should be 10 per page according to design

Reply:
Ah ok, I thought Cassidy had mentioned upping that now that we had pagination. But this can be an easy change in the future if we want!

Reply:
discussed offline with Design! think we overlooked the `items per page` option in the [LG/pagination component](https://www.mongodb.design/component/pagination/example/), so we can do a followup to allow items per page as well as page in the future

https://jira.mongodb.org/browse/DOP-3787?focusedCommentId=5648099&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-5648099

Reply:
Great, thanks!!